nmma/em/create_lightcurves.py

```diff
@@ -203,9 +202,9 @@ def get_parser():
     parser.add_argument(
         "--ylim",
         type=str,
-        default="-12,-18",
+        default="22,16",
```

Comment:
So the existing default's isn't bad for absolute magnitudes. Maybe we have a flag for --apparent or something that automatically switches the plots?

Reply:
TL:DR - I am bad at git 
This was done by @weizmannk some weeks ago, due to merge conflicts the commit related to this is reverted and showing up as new changes in this PR and few others changes as well. I will manually merge the his changes into this one

Reply:
looks like @weizmannk 's limits were wrong. ylim indeed should be positive, because lc is in absolute magnitude